tools/Google.Cloud.Tools.Analyzers/InternalOptionalParametersRequiredAnalyzer.cs

```diff
@@ -106,5 +85,38 @@ private static void AnalyzeInvocation(OperationAnalysisContext context)
                 }
             }
         }
+
+        internal static List<ISymbol> GetVariablesInScope(SyntaxNode syntaxNode, SemanticModel semanticModel) =>
```

Comment:
But the caller can call ToList if they want to. There's no point in calling ToList when you're then going to pass the result into Where and then call ToList on *that*, which is what you do elsewhere.